R/colour-text.r

```diff
@@ -5,9 +5,13 @@ testthat_colours <- list(
   error = crayon::red
 )
 
-colourise <- function(text, as = c("passed", "skipped", "error")) {
+#' @export
+colourise <- function(text, as = c("passed", "skipped", "error"), fg, bg) {
```

Comment:
This doesn't work for a call like `colourise("text", "green")`. OTOH, we really want to have `as` in the second position. @hadley: Any idea?